langserve/server.py

```diff
@@ -122,6 +123,45 @@ def _unpack_request_config(
     )
 
 
+def _update_config_with_defaults(
+    path: str, incomingConfig: RunnableConfig, endpoint: Optional[str] = None
+) -> RunnableConfig:
+    """Set up some baseline configuration for the underlying runnable."""
+
+    # Currently all defaults are non-overridable
+    overridable_default_config = RunnableConfig()
+
+    metadata = {}
+
+    if endpoint:
+        metadata["LangServe Endpoint"] = endpoint
+
+    is_hosted = os.environ.get("HOSTED_LANGSERVE_ENABLED", "false").lower() == "true"
+    if is_hosted:
+        hosted_metadata = {
+            "Commit SHA": os.environ.get("HOSTED_LANGSERVE_GIT_COMMIT", ""),
```

Comment:
The naming convention for these is different from langserve version etc. Should we use the same style? (I don't care which style is used)

Reply:
what is the current convention?

Reply:
just `__langserve`?

Reply:
yeah but whatever you prefer -- i just chose arbitrarily